### ISSUE TITLE
Go method and struct highlighting missing

### ISSUE THREAD

**Issue by 0x7061** [morhetz/gruvbox#36]
So I've been successfully using gruvbox on my iMac and wanted to also set it up in my vagrant machine running Ubuntu 14.04 LTS.
I wrote a provisioning file to install vim, vundle and gruvbox. Everything gets installed but if I compare the same file with my host OS X machine and the virtual machine with Ubuntu it looks different: 

![screen shot 2014-09-10 at 09 27 56](https://cloud.githubusercontent.com/assets/1425202/4213929/665fd7a0-38bc-11e4-85da-6e1704fc57a9.png)

Basically some highlights are missing. I'm also running `~/.vim/bundle/gruvbox/gruvbox_256palette.sh` in my `.bashrc`.

Any clue?
